### ISSUE TITLE
docker doesn't work

### ISSUE THREAD

**Issue by ghost** [DevSinghSachan/emdr2#7]

**Comment by DevSinghSachan** [comment]
Hey,
Could you paste the issue log here so that I can see it.

**Comment by ghost** [comment]
Yeah, sorry for the late reply! The error is related to swig telling me there's no -doxygen flag.

```shell
[ 77%] Swig compile swigfaiss.swig for python
swig error : Unrecognized option -doxygen
Use 'swig -help' for available options.
```

**Comment by DevSinghSachan** [comment]
Sorry for the late reply.

Looks like there is an error related to the FAISS installation. If you can find some other way to install FAISS, such as pip or conda, that should also work.

I had to install FAISS in this manner because, when I was doing experiments, FAISS binaries were not available for A100 GPUs. Hence, I had to manually compile them. But, now the situation might have changed. 

More recently, installing from conda forge might work well:
https://github.com/facebookresearch/faiss/blob/main/INSTALL.md#installing-from-conda-forge
